test/extensions/filters/http/cache/cache_filter_integration_test.cc

```diff
@@ -108,9 +108,14 @@ class CacheIntegrationTest : public Event::TestUsingSimulatedTime,
   OptRef<const Http::TestResponseTrailerMapImpl> empty_trailers_;
 };
 
-INSTANTIATE_TEST_SUITE_P(Protocols, CacheIntegrationTest,
-                         testing::ValuesIn(HttpProtocolIntegrationTest::getProtocolTestParams()),
-                         HttpProtocolIntegrationTest::protocolTestParamsToString);
+// Test for HTTP1/2 only.
+// TODO(asingh-g): Fix test suite for HTTP/3.
+INSTANTIATE_TEST_SUITE_P(
+    Protocols, CacheIntegrationTest,
+    testing::ValuesIn(HttpProtocolIntegrationTest::getProtocolTestParams(
+        /*downstream_protocols = */ {Http::CodecType::HTTP1, Http::CodecType::HTTP2},
+        /*upstream_protocols = */ {Http::CodecType::HTTP1, Http::CodecType::HTTP2})),
+    HttpProtocolIntegrationTest::protocolTestParamsToString);
```

Comment:
nit: how 'bout adding a helper method named something like `HttpProtocolIntegrationTest::getProtocolTestParamsWithoutHttp3()` to simplify doing this in all the various tests?